src/Page.test.jsx

```diff
@@ -0,0 +1,53 @@
+import { render, fireEvent } from '@testing-library/react';
+
+import Page from './Page';
+
+describe('App', () => {
+  it('renders title', () => {
+    const { container } = render(<Page tasks={[]} />);
+
+    expect(container).toHaveTextContent('To-do');
+    expect(container).toHaveTextContent('할 일');
+  });
+
+  it('calls handleChange', () => {
+    const handleChange = jest.fn();
+    const { getByRole } = render(
+      <Page
+        taskTitle="타이틀"
+        onChangeTitle={handleChange}
+        tasks={[{ id: 1, title: '일어나기' }]}
+      />
+    );
+
+    expect(handleChange).not.toBeCalled();
+    fireEvent.change(getByRole('textbox'), { target: { value: '세수하기' } });
+    expect(handleChange).toBeCalled();
```

Comment:
테스트 대상(When)과 결과 (Then)을 빈 줄로 구분해주면 좋을 것 같아요
```suggestion
    fireEvent.change(getByRole('textbox'), { target: { value: '세수하기' } });
    
    expect(handleChange).toBeCalled();
```

그리고 `not.toBeCalled`는 없어도 될 것 같아요

Reply:
감사합니다! 아직 `not.toBeCalled`를 사용하애 하는 부분에 대한 기준이 모호한데 혹시 명확한 기준이 있을까요?